### PR TITLE
Poking at poking at blobs

### PR DESCRIPTION
**Pull request by ghost**
Now no longer commit-fucked edition of #12252

Blob core cost now scales with the number of cores in existence (Currently set to +50 for each core in existence besides your own). To subside this, each node a core **places** _not takes over_ adds 10 additional blob point capacity, so a blob would have to make more nodes to be able to make more cores.

Blobs attacking supermatter now actually call explode on supermatter, rather than just making them disappear from existence.

Admins are now informed of who's saying what when it comes to blob telepathy. No more WGW

blob spores now inherit their factory DMI, so you can toss those in to a themed DMI

2 new blob skins - Honkscape (Courtesy of blithering)
blob_honk

and AME (Inspired by a bus celt did that one time)
blob_ame

Puts blob structure cost under defines for better management

More blob tweaks are followed if suggested

:cl:
- rscadd: Blob core cost now scales with the number of active blob cores in existence
- rscadd: Maximum blob points now scale with the number of nodes a blob core has CREATED
- rscadd: A few alternate blob designs are now available for admins to change blobs to
- rscadd: Blob spores now inherit the icon_state of the factory that creates them, opening up further customization options for blob
- rscadd: Admins are now given a proper log of who said what as a blob who is using telepathy
- rscdel: Removes the pulsing animation from 64x64 blobs, should reduce lag when blobs are active
- rscadd: Attacking supermatter as a blob now blows it the fuck up, rather than wiping it from existence

### CODE REVIEW

File: vgstation13.dme [hunk 1]
```diff
@@ -2053,7 +2054,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```
Comment (Intigracy) [inline vgstation13.dme]: yuck

Comment (ghost) [reply vgstation13.dme]: fuck

**Comment by NigglyWiggly** [comment]
Fuuuuck this might mean I need to finish my cute blob
blob

**Comment by Exxion** [comment]
The shading in some of the areas in the bottom right is a war crime

**Comment by NigglyWiggly** [comment]
Yeah, I made all the little tiles of it, and then this is what it looked like when I assembled it all together in a little test blob.



<sub>if you look carefully you can find an area where i cheated because the tiles didn't line up very well

**Comment by ghost** [comment]
Does anything else need changing with this?

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**